int16[] ids
sensor_msgs/PointCloud[] clouds
